po/uk.po

```diff
@@ -2002,114 +2002,116 @@ msgstr ""
 
 #: wt-status.c:242
 msgid "  (use \"git add/rm <file>...\" to update what will be committed)"
-msgstr ""
+msgstr "  (використовуйте \"git add/rm <файл>…\" для оновлення того, що буде закомічено)"
```

Comment:
```suggestion
msgstr "  (використовуйте \"git add/rm <файл>...\" для оновлення того, що буде закомічено)"
```

Reply:
I'm not sure what happened to those... Thanks!